apcd_cms/src/apps/utils/apcd_database.py

```diff
@@ -1281,7 +1281,7 @@ def get_current_exp_date(submitter_id, applicable_data_period):
         if conn is not None:
             conn.close()
 
-def get_all_extensions():
+def get_all_extensions(submitter_codes=None):
     cur = None
     conn = None
     try:
```

Comment:
Sorry I didn't catch this earlier. Maybe we can replace line 1288-1295 with:

` conn = db_connect()`

Totally optional just for cleanup